whelktool/scripts/cleanups/2022/01/lxl-3589-link-agents.groovy

```diff
@@ -94,14 +96,20 @@ Map loadAgents() {
                 agents[chip] << id
             else
                 agents[chip] = [id] as Set
+
+            chip = chip.findAll { it.key != '@type' }
+            if (agents.containsKey(chip))
+                agents[chip] << id
+            else
+                agents[chip] = [id] as Set
+
         }
```

Comment:
Do you really want to store one agent with '@type' and one without?

Reply:
One could probably come up with a more clever solution, I just implemented the first somewhat (time) efficient solution that came to mind :)

Reply:
Sure, if it works the way you planned it's perfectly OK!